mobile-ui/src/main/java/com/playone/mobile/ui/firebase/v1/PlayoneFirebaseV1.kt

```diff
@@ -168,7 +424,7 @@ class PlayoneFirebaseV1(
                                 ::snapToPlayone)
         }
     }
-    //endregion
+//endregion
```

Comment:
actually I don't really like this kind of comments (it is pointless for me), what do you think?

Reply:
I don't like it, either. __BUT__ when I open this file, it's difficult to find a method what I want. :( 
How do you fix kind of the problems? Please tell me, bro :(

Reply:
You can use the help by IDE. In Android Studio, the left panel there is `Structure`, I often use that to find the function I am looking for if the file is big.

Reply:
That's a good idea for finding a method you want to find. But for some parts, I hope we can keep them because IDE can do collapse for us.